library/ix-dev/community/rsyncd/values.yaml

```diff
@@ -0,0 +1,19 @@
+# FIXME: Switch to stable tag
+image:
+  repository: ixsystems/rsyncd
+  pullPolicy: IfNotPresent
+  tag: 'unstable'
```

Comment:
I assume you want to change this to `1.0.0`

Reply:
Correct